src/pages/game/Game.tsx

```diff
@@ -1,8 +1,16 @@
 import React, { FC, ReactElement } from 'react';
 import { Link } from 'react-router-dom';
 
-import { routeConstants } from 'constants/routeConstants';
+import GameCanvas from 'components/gameCanvas/GameCanvas';
+import { ROUTE_CONSTANTS } from 'constants/routeConstants';
 
-const Game: FC = (): ReactElement => <Link to={routeConstants.DASHBOARD}>Go to dashboard</Link>;
+const Game: FC = (): ReactElement => {
+  return (
+    <div>
+      <GameCanvas></GameCanvas>
```

Comment:
❗️
Всегда юзай самозакрывающиеся теги, где нет детей. В еслинте есть хорошее правило на этот счёт
```suggestion
      <GameCanvas />
```